### ISSUE TITLE
How to use "Notification props: component"

### ISSUE THREAD

**Issue by rushairer** [creativetimofficial/ct-vue-argon-dashboard-pro#7]
!!! IF YOU DO NOT USE THIS ISSUES TEAMPLATE, YOUR ISSUE IS LIABLE TO BEING IGNORED BY US

# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (for multi-repository projects)

# Expected Behavior

I want to use 'Notifications' with props 'component'

# Current Behavior
# Failure Information (for bugs)
## Context

How can I use  'component', is there a demo? Thanks.

## Failure Logs

**Comment by rushairer** [comment]
I have changed some code.

NotificationPlugin\Notification.vue
change 
```js
components: {
        contentRender: {
            props: ['component'],
            render: h => h(this.component)
        }
    },
```
to
```js
components: {
        contentRender: {
            props: ['component'],
            render: function (createElement) {
                return createElement(this.component)
            }
        }
    },
```

Because the vue document said:
https://vuejs.org/v2/guide/instance.html#Instance-Lifecycle-Hooks
```
Don’t use arrow functions on an options property or callback, 
such as created: () => console.log(this.a) or vm.$watch('a', newValue => this.myMethod()). 
Since an arrow function doesn’t have a this, this will be treated as any 
other variable and lexically looked up through parent scopes until found,
 often resulting in errors such as Uncaught TypeError: Cannot read property
 of undefined or Uncaught TypeError: this.myMethod is not a function.

```

**Comment by cristijora** [comment]
Thanks for mentioning the issue @rushairer We've updated the code to this version as well!

**Comment by cristijora** [comment]
Released v1.1.1 which contains this fix
